spec/classes/foreman_proxy_content_spec.rb

```diff
@@ -71,6 +71,40 @@ class { 'foreman_proxy::plugin::pulp':
         end
       end
 
+      context 'pulp-2to3-migration' do
+        let(:params) do
+          {
+            qpid_router: false
+          }
+        end
+
+        let(:pre_condition) do
+          <<-PUPPET
+          include foreman_proxy
+          include katello::pulp
```

Comment:
This shouldn't be needed since the code also has an include
```suggestion
```

Reply:
@ekohl in fact I had to remove that include since it was causing conflicts when both pulp2 and pulpcore are enabled (we are already declaring pulp). I'm now getting the mongo params from the already declared pulp class. I'll remove the dependencies for puppet-katello now since they are no longer needed.